index.html

```diff
@@ -913,6 +952,10 @@ <h3>
           <a data-cite="FETCH#concept-request">request</a>.
           </li>
           <li>If <var>request</var>'s <a data-cite=
+          "FETCH#concept-request-redirect-count">redirect count</a> is zero, return
```

Comment:
This does not seem quite right as this check seems to also involve the 'previous document' (i.e. the document being unloaded, since unload values rely on this check).

Reply:
Sorry I'm not following -- the "previous document" check is at Step (1), which would return `fail`.  Everything after is either because it's coming from no previous document or the previous document was the same origin, and Steps (2-5) just looks at the current document's redirects.